c/doc/run_sample_on_intel_edison.md

```diff
@@ -121,12 +161,12 @@ The result should be the following:
 # ./iothub_client_sample_amqp
 hub_client/samples/iothub_client_sample_amqp/linux#
 Starting the IoTHub client sample AMQP...
-IoTHubClient_SetMessageCallback...successful.
-IoTHubClient_SendEventAsync accepted data for transmission to IoT Hub.
-IoTHubClient_SendEventAsync accepted data for transmission to IoT Hub.
-IoTHubClient_SendEventAsync accepted data for transmission to IoT Hub.
-IoTHubClient_SendEventAsync accepted data for transmission to IoT Hub.
-IoTHubClient_SendEventAsync accepted data for transmission to IoT Hub.
+IoTHubClient_SetNotificationCallback...successful.
```

Comment:
Why has this changed? The sample prints out IoTHubClient_SetMessageCallback & IoTHubClient_SendEventAsync. Looks like this is old.